msdsales/methods.go

```diff
@@ -0,0 +1,129 @@
+package msdsales
+
+import (
+	"context"
+	"errors"
+	"log/slog"
+
+	"github.com/amp-labs/connectors/common"
+	"github.com/amp-labs/connectors/common/reqrepeater"
+)
+
+// FIXME these methods look repetitive
+// FIXME arguments: Retry Strategy + HTTP Client.
```

Comment:
Every CRUD method is identical to other connectors. This could be refactored/generated.

Reply:
Can you replace `FIXME` with `TODO`? then it's easier for us to do to a search for TODOs across the codebase at a later point.